src/melee/it/types.h

```diff
@@ -262,18 +215,25 @@ struct ItemAttr {
 };
 
 struct ItemDynamicsDesc {
-    s32 x0_boneID;   // bone index;
-    void* x4_params; // dynamics params;
-    s32 x8_num;      // number of children bones to make dynamic
+    int x0_boneID;   // bone index;
+    unk_t x4_params; // dynamics params;
+
+    /// @at{8} @sz{4}
+    /// @brief Number of children bones to make dynamic.
```

Comment:
We decided on out-of-line docs midway through the creation of this PR so I'll move these over later.